_data/osdf_namespace_metadata/mu-ahurt-public.yaml

```diff
@@ -1,15 +1,15 @@
 description: |
 
-organization: # Name of the organization
-dataVisibility: # public, or private
+organization: University of Missouri # Name of the organization
+dataVisibility: public # public, or private
 size: # In bytes so I can parse how I want
 bytesXferd: # In bytes so I can parse how I want
-url: # Organization URL
-fieldOfScience: # Top level Field of Science pulled from here https://nces.ed.gov/ipeds/cipcode/browse.aspx?y=55
+url: https://missouri.edu/ # Organization URL
+fieldOfScience: 11 # Top level Field of Science pulled from here https://nces.ed.gov/ipeds/cipcode/browse.aspx?y=55
```

Comment:
I think we want COMPUTER AND INFORMATION SCIENCES AND SUPPORT SERVICES for this one